packages/cli/src/wiring/CliInitializer.ts

```diff
@@ -14,46 +14,71 @@
  * limitations under the License.
  */
 
-import { CommandGraph } from './CommandGraph';
-import { CliFeature, InternalCliFeature, InternalCliPlugin } from './types';
-import { CommandRegistry } from './CommandRegistry';
+import { CommandGraph } from './commands/CommandGraph';
+import { InternalFeature } from './types';
 import { program } from 'commander';
 import { version } from '../lib/version';
 import chalk from 'chalk';
 import { exitWithError } from '../lib/errors';
 import { assertError } from '@backstage/errors';
+import { ServiceRegistry } from './services/ServiceRegistry';
+import { CliFeature } from './features/types';
+import { createServiceFactory, InternalServiceFactory } from './services/types';
```

Comment:
Maybe adding `cli` to the naming patterns? `createClieServiceFactory` etc. Or some new naming that's not "service" or "api"; the frontend/backend separation is based on inventing a new name after all :P (not suggesting this is the best solution though, but some bikeshedding welcome. Sprocket?)

Reply:
hmmm, could we use some framework sig time for brainstorming? I like the idea of a new set of terminology, writing `cli` feels really clunky for some reason, maybe even `terminal` instead would be better.